api/tests/unit/segments/test_unit_segments_views.py

```diff
@@ -579,6 +580,78 @@ def test_update_segment_delete_existing_rule(project, client, segment, segment_r
     assert segment_rule.conditions.count() == 0
 
 
+@pytest.mark.parametrize(
+    "client",
+    [lazy_fixture("admin_master_api_key_client"), lazy_fixture("admin_client")],
+)
+def test_create_segment_with_required_metadata_returns_201(
```

Comment:
We should also add test where metadata filed for segments is required across organisation and not just project (we support that, right? )
ps: we should do the same for features as well

Reply:
Done.